File: C/1.Iniciante/1117.c
#include <stdio.h>

int main(void)
{
    float nota1, nota2;

    Nota1:
    scanf("%f", &nota1);
    if(nota1<0 || nota1>10.0){
        printf("nota invalida\n");
        goto Nota1;
    }

    Nota2:
    scanf("%f", &nota2);
    if(nota2<0 || nota2>10.0){
        printf("nota invalida\n");
        goto Nota2;
    }
    printf("media = %.2f\n", (nota1+nota2)/2);
    return 0;
}

File: 1041/1117.py
soma = 0
media = 0
i = 0
while True:
    x = float(input())

    if x < 0 or x > 10:
        print('nota invalida')

    elif 0 <= x <=10:
        soma += x
        i += 1

    if i == 2:
        break

media = soma / i
print(f'media = {media:.2f}')